server/utils/urlGenerator.ts

```diff
@@ -118,6 +118,9 @@ const adjudicationUrls = {
   },
   yourCompletedReports: {
     root: '/your-completed-reports',
+    urls: {
```

Comment:
is this needed? I can't see where it is used

Reply:
I think you are correct - I'll remove

Reply:
Ah - actually its used in the njk file